### PR TITLE
docs: Minor tweak to attribute docs.

### PR DESCRIPTION
**Pull request by vpetrovykh**
Clarify that attributes can only be of type `str`.

Make the use of quotes consistent in text and examples for attribute
values.

### CODE REVIEW

File: docs/edgeql/ddl/attributes.rst [hunk 1]
```diff
@@ -118,7 +118,7 @@ Create an object type ``User`` and set its ``title`` attribute to
 .. code-block:: edgeql
 
     CREATE TYPE User {
-        SET ATTRIBUTE title := 'User type';
+        SET ATTRIBUTE title := "User type";
```
Comment (1st1) [inline docs/edgeql/ddl/attributes.rst]: Why this change?

Comment (vpetrovykh) [reply docs/edgeql/ddl/attributes.rst]: To be consistent with the text above the example.
